packages/login-with-reapit/README.md

```diff
@@ -0,0 +1,41 @@
+# Login With Reapit
+
+
+## Install
+
+```bash
+$ yarn add @reapit/login-with-reapit
+```
+
+```bash
+$ npm i @reapit/login-with-reapit
+```
+
+## Usage
+
+```ts
+import { reapitConnectComponent } from '@reapit/login-with-reapit'
+
+const connectHasSessionCallback = (reapitConnectBrowserSession) => {
+  reapitConnectBrowserSession.connectSession().then(session => {
+    console.log('Session is', session)
+  })
+}
+
+reapitConnectComponent({
+  connectClientId: '', // Your client Id
+  connectOAuthUrl: 'https://connect.reapit.cloud', // Our OAuth url
+  connectLoginRedirectPath: '', // Your logout page path (when logging in)
+  connectLogoutRedirectPath: '/login', // Your login page path (when logging out)
+  rootElement: document.querySelector('#reapit-connect-component'), // Your Element on the page
+  connectHasSessionCallback, // The callback for when the session is created
+  companyName: "Test Company", // Your Company name
+})
+```
+
+## Shadow Root Element
```

Comment:
did you mean to finish this or is that coming in another PR?

Reply:
yes, I need to work this bit out. Now I've built the package I can deploy + test it out